server/controllers/apply/withdrawablesController.ts

```diff
@@ -0,0 +1,20 @@
+import type { Request, RequestHandler, Response } from 'express'
+import { ApplicationService } from '../../services'
+import paths from '../../paths/apply'
+import { ApprovedPremisesApplication as Application } from '../../@types/shared'
+
+export default class WithdrawalsController {
+  constructor(private readonly applicationService: ApplicationService) {}
+
+  show(): RequestHandler {
```

Comment:
I think this logic is better sitting in the `WithdrawlsController#new` controller action. If there are multiple withdrawls, or `withdrawlType` is not set, we show the withdrawables. If the user selects the application, we set `withdrawlType` to `application` and we show the withdrawal options for the application.